ext/pdo/pdo_sql_parser.re

```diff
@@ -259,6 +259,10 @@ safe:
 					switch (param_type) {
 						case PDO_PARAM_BOOL:
 							plc->quoted = zend_is_true(parameter) ? ZSTR_CHAR('1') : ZSTR_CHAR('0');
+							// TODO Can an exception occur here?
+							if (EG(exception)) {
```

Comment:
How about adding `ret = -1;` here to at least stop the execution?

Reply:
That's probably even necessary, been a while since I looked at this code.